src/bsp/crypto/iotc_bsp_crypto_test_ecc.cc

```diff
@@ -0,0 +1,186 @@
+/* Copyright 2018-2019 Google LLC
+ *
+ * This is part of the Google Cloud IoT Device SDK for Embedded C,
+ * it is licensed under the BSD 3-Clause license; you may not use this file
+ * except in compliance with the License.
+ *
+ * You may obtain a copy of the License at:
+ *  https://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cstring>
+#include <iostream>
+
+#include "gtest.h"
+#include "iotc.h"
+#include "iotc_bsp_crypto.h"
+#include "iotc_heapcheck_test.h"
+#include "iotc_helpers.h"
+#include "iotc_jwt.h"
+#include "iotc_macros.h"
+#include "iotc_openssl_utils.h"
+#include "iotc_types.h"
+
+namespace iotctest {
+namespace {
+
+constexpr char kPublicKey[] = "\
```

Comment:
Could you use raw string literals for private and public keys? That would improve readability.

https://abseil.io/tips/64